UnrealPlugin/README.md

```diff
@@ -1,4 +1,5 @@
 # Instructions on Integrating the PlayFab GSDK Unreal Plugin Preview
+
```

Comment:
Note: Extra spaces in the link are added in the comments so it's easier to copy and paste in markdown. Remove them on your own.

Reply:
Understood thanks for the clarification.